.obs/chartfile/operator/values.yaml

```diff
@@ -10,8 +10,9 @@ seedImage:
   imagePullPolicy: IfNotPresent
 
 channel:
-  image: "%%IMG_REPO%%/rancher/elemental-channel"
-  tag: "%VERSION%"
+  name: "sle-micro-5.5-channel"
+  image: "%%IMG_REPO%%/rancher/elemental-channel/sle-micro"
```

Comment:
Also if we have development channel, there is no reason to have this `%%IMG_REPO%%` as this should be an official already existing channel reference such as `registry.suse.com/rancher/elemental-channel/...`

Reply:
cannot agree more 👍🏼